### ISSUE TITLE
Login bug

### ISSUE THREAD

**Issue by scottofletcher** [participedia/frontend#801]
From Dr. Warren's daughter (computer coder):

"Bug repro steps:

1. Click on "Quick Submit"
2. Exit out of pop up (notice url still has /new at the end)

Steps to see login modal continuously reappear:
3. Click on login
4. Exit out of login
5. Click on menu"

Scott's note: I have no idea what any of that means. I assume it makes sense to you folks!

**Comment by dethe** [comment]
It seems to mean that if you try to get to a page that requires you to login, but close the login popup without logging in, it keeps asking you to login if you stay on that page. Not sure what the problem being reported is, that seems like the right behaviour to me.

**Comment by scottofletcher** [comment]
Yeah, I agree with Dethe - if you click Quick Submit or Login, it would make sense for the popup to re-appear. I assume that it stops doing that if you refresh or navigate away (ie. if you clicked quick submit or login in error). Regardless, this might actually be a good feature since people will have to login or signup if they want to continue (thus increasing our number of active users!)

**Comment by scottofletcher** [comment]
I think we should close this - if it's not an issue we shouldn't be worrying about it
